dev/bin/configure.sh

```diff
@@ -93,12 +118,23 @@ function configure_minikube {
 
 function configure_minikube_started {
   configure_minikube
-  MINIKUBE_IP=$( minikube -p "${MINIKUBE_PROFILE}" ip ) || die "can't find minikube ip. Is it started?"
-  ping -c 1 "${MINIKUBE_IP}" || die "minikube is not responding to ping. Is it started?"
+  CLUSTER_IP=$( minikube -p "${MINIKUBE_PROFILE}" ip ) || die "can't find minikube ip. Is it started?"
+  ping -c 1 "${CLUSTER_IP}" || die "minikube is not responding to ping. Is it started?"
+}
+
+function configure_kind {
+  KIND_CONFIG_FILE="${KIND_CONFIG_FILE:-"${DEV_CONFIG_DIR}/kind/config.yaml"}"
+  KIND_KUBERNETES_VERSION="${KIND_KUBERNETES_VERSION:-v1.20.0}"
+}
+
+function configure_kind_started {
+  configure_kind
+  CLUSTER_IP='localhost'
+  ping -c 1 "${CLUSTER_IP}" || die "kind is not responding to ping. Is it started?"
 }
 
 function getKeycloakAccessToken {
```

Comment:
I think we should uniform the function names to snake case instead of mixing it with camel case

Reply:
you are right. gonna update those methods

Reply:
done